ui_tests/test_ui_inputs.py

```diff
@@ -2872,6 +2872,7 @@ def test_save_to_python(self):
             try:
                 spec = importlib.util.spec_from_file_location(module_name, python_file)
                 module = importlib.util.module_from_spec(spec)
+                spec.loader.exec_module(module)
```

Comment:
We've been skipping this whole class of tests for a while, but I learned when I copied this test to `test_cli.py` that this line is now necessary.